pages/art.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Art</title>
+  <link href="/styles/style.css" rel="stylesheet" />
+</head>
+<body class="container">
+  <header>
+    <h1 class="header">ART</h1>
+    <nav class="nav">
+      <ul>
+        <li><a href="index.html">HOME</a></li>
+        <li><a href="about.html">ABOUT</a></li>
+        <li><a href="portfolio.html">PORTFOLIO</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <section class="art-container">
+      <a href="/images/tiefling.jpg"><img src="/images/tiefling_cropped.png" alt="illustration of a red tiefling with white-and-orange hair." width="200"></a>
+      <a href="/images/roda.jpg"><img src="/images/roda_cropped.png" alt="rough sketch of a pretty short-haired boy in a white mage outfit a la final fantasy." width="200"></a>
+      <a href="/images/ruth.jpg"><img src="/images/ruth_cropped.png" alt="monochromatic image of a woman in a hunter's outfit, palm resting on the pommel of a blood-spattered blade." width="200"></a>
+      <a href="/images/juniper.jpg"><img src="/images/juniper_cropped.png" alt="simple colored art of a smiling androgynous person with red glasses and messy violet hair" width="200"></a>
+      <a href="/images/inauno.jpg"><img src="/images/inauno_cropped.png" alt="painted illustration of two elf women, 
+        one grey-skinned and dressed in dull tones, one green-skinned and dressed in bright colors. they're both smiling and holding hands." width="200"></a>
+      <a href="/images/lyna.jpg"><img src="/images/lyna_cropped.png" alt="illustration of a rabbit-eared woman in fantasy-styled armor with a red cape. 
+        she holds a salute with her fists held close to her chest." width="200"></a>
+    </section>
```

Comment:
Style: Since this doesn't have a semantic meaning it should just be a `div` (or styles applied directly to `main`) not a `section`.  (That's why the validator warned you about not having a heading.)